README.md

```diff
@@ -242,7 +242,7 @@ Make sure that you have the following required dependencies installed:
 
 * CMake v3.5.1 or higher
 
-* [Go](https://golang.org/) 1.11.x or higher
+* [Go](https://golang.org/) 1.14.x or higher
```

Comment:
1.14 required as minimum or only as the newest thing available at the time?

Reply:
Fabric v2 requires >= 1.13.4; i assume Marcus just bumped to 1.14 as it's already out and thought we might as well require that to be more future proof?